reddit_decider/__init__.py

```diff
@@ -99,6 +102,7 @@ def to_event_dict(self) -> Dict:
             "id": self._user_id,
             "logged_in": self._logged_in,
             "cookie_created_timestamp": self._cookie_created_timestamp,
+            "loid_created_timstamp": self._loid_created_timestamp,
```

Comment:
this function is used to dump data for v2 event exposures, and the v2 event schema doesn't have this field on [user](https://github.snooguts.net/reddit/data-schemas/blob/master/schemas/components/user.thrift) so we can eliminate it here